_benchmark/user-guide/understanding-results.md

```diff
@@ -0,0 +1,125 @@
+|                                                 Max Throughput |                   wait-until-merges-finish |       28.41 |  ops/s |
+|                                       100th percentile latency |                   wait-until-merges-finish |     34.7088 |     ms |
+|                                  100th percentile service time |                   wait-until-merges-finish |     34.7088 |     ms |
+|                                                     error rate |                   wait-until-merges-finish |           0 |      % |
+|                                                 Min Throughput |                                  match_all |       36.09 |  ops/s |
+|                                                Mean Throughput |                                  match_all |       36.09 |  ops/s |
+|                                              Median Throughput |                                  match_all |       36.09 |  ops/s |
+|                                                 Max Throughput |                                  match_all |       36.09 |  ops/s |
+|                                       100th percentile latency |                                  match_all |     35.9822 |     ms |
+|                                  100th percentile service time |                                  match_all |     7.93048 |     ms |
+|                                                     error rate |                                  match_all |           0 |      % |
+
+[...]
+
+|                                                 Min Throughput |                                       term |        16.1 |  ops/s |
+|                                                Mean Throughput |                                       term |        16.1 |  ops/s |
+|                                              Median Throughput |                                       term |        16.1 |  ops/s |
+|                                                 Max Throughput |                                       term |        16.1 |  ops/s |
+|                                       100th percentile latency |                                       term |     131.798 |     ms |
+|                                  100th percentile service time |                                       term |     69.5237 |     ms |
+|                                                     error rate |                                       term |           0 |      % |
+```
+
+The unique metrics to yhe cluster begin at the `index` task line. The following two use cases can give you an idea of what metrics might be relevant to you:
+
+- To assess how much load your cluster can handle, the `index` task metrics provide the number of documents ingested during the workload run. 
+- To assess how fast you can query against documents in your cluster, the `match_all` and `term` give both the number of query operations performed per second and the measurable latency of the query.
```

Comment:
the measurable latency and service time of the query

Reply:
Maybe add additional point about error rate associated with the ingesting documents and running query operations should be added to both points.